inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/prior_box.cpp

```diff
@@ -50,8 +50,17 @@ const std::vector<std::vector<float>> variances = {
 const std::vector<bool> scale_all_sizes = {
     false, true};
 
-const std::vector<size_t> inputShape = {300, 300};
-const std::vector<size_t> imageShape = {32, 32};
+const std::vector<ov::test::InputShape> inputShape = {
```

Comment:
expand number of operation specific attributes
most of them empty at this moment

Reply:
tests parameters were not updated, previous meaningful values are used. I would prefer extend tests during another task, not dynamism